src/libraries/EndpointStructs.sol

```diff
@@ -16,11 +16,8 @@ struct EndpointManagerMessage {
 struct NativeTokenTransfer {
     /// @notice Amount being transferred (big-endian uint256)
     uint256 amount;
-    /// @notice Address of the token. Left-zero-padded if shorter than 32 bytes
-    // TODO: this field is not needed.
-    bytes32 tokenAddress;
-    /// @notice Address of the recipient. Left-zero-padded if shorter than 32 bytes
-    bytes32 to;
+    /// @notice Address of the recipient.
+    bytes to;
```

Comment:
Nice adding the payload and recipient address lengths. Perhaps we should add them in the structs as well?

I imagine people will reference these structs when building additional services -- so add `NativeTokenTransfer.recipientLength` and `EndpointManagerMessage.payloadLength`?

Reply:
That's a good point. I think we should instead include a comment explaining what the wire format looks like -- I don't feel very good adding that field into the struct because that introduces newly representable invalid states that we then have to think about (for example one could create a struct where the length field doesn't actually match the length of the address)